styles/main.scss

```diff
@@ -354,7 +354,12 @@ select.search-input:disabled {
 
 .results-cell p {
     margin: 0 0 .5em 0;
+    font-size: 16px;
     @include text-ellipses-overflow;
+
+    @include for-tablet-portrait-up {
+        font-size: unset;
```

Comment:
not a css expert, but we don't want to be explicit about the font size?

Reply:
I can see how this is misleading - I was actually unsetting so that my specification of the font size on the `.results-cell` class would kick in for tablet portrait up. I actually think it's easier to just take care of all the font-size specification on the `.results-cell` selector, which means we don't need this unsetting clause and it's a little cleaner.